go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -36,6 +37,13 @@ func gatewayCleanup(nodeName string) error {
 			"stderr: %q, error: %v", types.JoinSwitchToGWRouterPrefix, gatewayRouter, stderr, err)
 	}
 
+	// Remove router to lb associations from the LBCache before removing the router
+	lbCache, err := ovnlb.GetLBCache()
+	if err != nil {
+		return fmt.Errorf("failed to get load_balancer cache for router %s: %v", gatewayRouter, err)
```

Comment:
In this case, I'd swallow the error and move on.

Reply:
should we? I thought so too, but then we'd hit the other "ls-lb-del" bug and transaction would fail. That triggers a whole bunch of problems.